translations/base-de.yaml

```diff
@@ -776,7 +776,7 @@ keybindings:
 
         rotateWhilePlacing: Rotieren
         rotateInverseModifier: >-
-            Modifier: stattdessen gegen UZS rotieren
+            Modifikator: stattdessen gegen UZS rotieren
```

Comment:
Hier hast du den Schlüssel selbst (`Modifier`) auch übersetzt

Reply:
![grafik](https://user-images.githubusercontent.com/57270769/85313676-ab59e980-b4b8-11ea-851b-b4c7e56d9b1a.png)
Ich wollte diesen Teil hier in den Einstellungen für die Tastenbelegung übersetzen. Ist das (woanders) möglich/muss ich dort in der Formatierung etwas ändern oder die ganze Änderung zurücknehmen?

Reply:
Ahh, mein Fehler. Alles gut, hatte das einfach überlesen :)